docs/faq.md

```diff
@@ -643,3 +643,24 @@ force colors on by setting `FORCE_COLOR` to any non-empty value (e.g. `FORCE_COL
 
 [`colored`](https://crates.io/crates/colored) also supports the `CLICOLOR` and `CLICOLOR_FORCE`
 environment variables (see the [spec](https://bixense.com/clicolors/)).
+
+## Ruff behaves unexpectedly when using `source.*` code actions in Notebooks. What's going on?
+
+Starting with `0.9.7`, Ruff does not support `source.organizeImports` and `source.fixAll` code
```

Comment:
Not sure if we want to highlight the version as we should just recommend users not to use it regardless of the version.

I think I'll remove it.

Reply:
Agree, I'd remove the version because it also applies to older versions